client-device-connection-infinispan/pom.xml

```diff
@@ -22,6 +22,10 @@
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-client-hotrod</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-core</artifactId>
+    </dependency>
```

Comment:
For actually using the embedded cache mode in a kubernetes cluster, we would also need the `jgroups-kubernetes` dependency, wouldn't we? Or do you want to add that later, along with maybe an example config?

Reply:
I would do that outside of the scope of this PR.